formbricks/formbricks
# feat: added in-page section navigation for docs

## What does this PR do?



Fixes #2718

![image](https://github.com/formbricks/formbricks/assets/56228323/cde909e9-0a49-40ad-b379-e80193695ec4)



## How should this be tested?



- Start the dev server
- Visit http://localhost:3001/docs/developer-docs/contributing
- Check the page section navigation on the right

## Checklist



### Required


### Appreciated
